libs/libarchfpga/src/read_xml_arch_file_interposer.cpp

```diff
@@ -0,0 +1,44 @@
+#include "read_xml_arch_file_interposer.h"
+#include <vector>
+#include "interposer_types.h"
+#include "read_xml_util.h"
+#include "pugixml_util.hpp"
+#include "arch_error.h"
+
+t_interposer_cut_inf parse_interposer_cut_tag(pugi::xml_node interposer_cut_tag, const pugiutil::loc_data& loc_data) {
+    t_interposer_cut_inf interposer;
+
+    pugiutil::expect_only_attributes(interposer_cut_tag, {"dim", "loc"}, loc_data);
+
+    std::string interposer_dim = pugiutil::get_attribute(interposer_cut_tag, "dim", loc_data).as_string();
+    if (interposer_dim.size() != 1 || !CHAR_INTERPOSER_DIM_MAP.contains(interposer_dim[0])) {
+        archfpga_throw(loc_data.filename_c_str(), loc_data.line(interposer_cut_tag), "Interposer tag dimension must be a single character of either X, x, Y or y.");
+    }
+
+    interposer.dim = CHAR_INTERPOSER_DIM_MAP.at(interposer_dim[0]);
+
+    interposer.loc = pugiutil::get_attribute(interposer_cut_tag, "loc", loc_data).as_int();
+    if (interposer.loc < 0) {
```

Comment:
What does it mean if the interposer location is 0?  Goes back to the question of which side of an interposer the specified value goes on.  If the lower side, then a loc of 0 makes sense. Otherwise it doesn't make sense.